source/reference/auth-plugin-c.txt

```diff
@@ -158,63 +180,101 @@ ODBC Connection Parameter
 If you are using the MySQL ODBC driver, the interface you use to
 configure your :abbr:`DSN (Data Source Name)` may provide a
 field where you can specify the default authentication
-plugin to use. Specifying ``mongosql_auth`` here will cause the
+plugin to use. Specifying ``mongosql_auth`` in that field causes the
 ODBC driver to use the ``mongosql_auth`` plugin by default.
  
 .. _mysql-config: 
 
 Configuration File
 ~~~~~~~~~~~~~~~~~~
 
-MySQL configuration files can be found in many locations, as listed
-in the `MySQL documentation
-<https://dev.mysql.com/doc/refman/5.7/en/option-files.html>`_.
-In one of these files, add a line with ``default-auth=mongosql_auth`` to
-the ``[client]`` section or create it if it does not yet exist.
-To use this same configuration file with an ODBC :abbr:`DSN
-(Data Source Name)`, provide the ``USE_MYCNF=1`` connection parameter
-to your ODBC :abbr:`DSN (Data Source Name)`.
+MySQL configuration files can be found in many locations, as listed in
+the `MySQL documentation <https://dev.mysql.com/doc/refman/5.7/en/option-files.html>`_. 
+In one of these files, add a line with ``default-auth=mongosql_auth`` 
+to the ``[client]`` section. Create the ``[client]`` section if it does not exist. 
+
+To use this configuration file with an
+:abbr:`ODBC (Open Database Connectivity)` 
+:abbr:`DSN (Data Source Name)`, add the ``USE_MYCNF=1`` connection 
+parameter to your :abbr:`ODBC (Open Database Connectivity)`
+:abbr:`DSN (Data Source Name)`.
```

Comment:
Just verifying: the ``USE_MYCNF=1`` parameter should be added to the ODBC DSN and not the configuration file, correct?

Reply:
correct -- that is an ODBC connection parameter that tells the ODBC driver to load the configuration options specified in my.cnf